pkg/gui/controllers/files_controller.go

```diff
@@ -561,14 +561,21 @@ func (self *FilesController) createStashMenu() error {
 				OnPress: func() error {
 					return self.handleStashSave(self.git.Stash.Save, self.c.Tr.Actions.StashAllChanges)
 				},
-				Key: 's',
+				Key: 'a',
 			},
 			{
 				DisplayString: self.c.Tr.LcStashStagedChanges,
 				OnPress: func() error {
-					return self.handleStashSave(self.git.Stash.SaveStagedChanges, self.c.Tr.Actions.StashStagedChanges)
+					return self.handleStagedStashSave(self.git.Stash.SaveStagedChanges, self.c.Tr.Actions.StashStagedChanges)
+				},
+				Key: 's',
+			},
+			{
+				DisplayString: self.c.Tr.LcStashUnstagedChanges,
+				OnPress: func() error {
+					return self.handleStashSave(self.git.Stash.SaveUnstagedChanges, self.c.Tr.Actions.StashUnstagedChanges)
```

Comment:
there's no need to pass in these two arguments here, given that the method itself knows that we want to stash staged changed. I would instead pass them in to `handleStashSave` explicitly on line 618